source/renderer/app/components/staking/redeem-itn-rewards/NoWalletsDialog.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
+import SVGInline from 'react-svg-inline';
+import DialogCloseButton from '../../widgets/DialogCloseButton';
+import Dialog from '../../widgets/Dialog';
+import styles from './Step3FailureDialog.scss';
+import redeemDialogOverride from './RedeemDialogOverride.scss';
+import sadWalletImage from '../../../assets/images/sad-wallet.inline.svg';
+
+const messages = defineMessages({
+  description: {
+    id: 'staking.redeemItnRewards.noWallets.description',
+    defaultMessage:
+      '!!!Redemption of Incentivized Testnet rewards is not available as you currently do not have any Shelley-compatible wallets.',
+    description: 'description for Redeem Incentivized Testnet - Step 3',
+  },
+  addWalletButtonLabel: {
+    id: 'staking.redeemItnRewards.noWallets.addWalletButtonLabel',
+    defaultMessage: '!!!Add wallet',
+    description:
+      'addWalletButtonLabel for Redeem Incentivized Testnet - Step 3',
+  },
+});
+
+type Props = {
+  onClose: Function,
+  onAddWallet: Function,
+};
+
+@observer
+export default class NoWalletsDialog extends Component<Props> {
+  static contextTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  render() {
+    const { intl } = this.context;
+    const { onClose, onAddWallet } = this.props;
+    return (
+      <>
+        <DialogCloseButton
+          className={redeemDialogOverride.closeButton}
+          onClose={onClose}
+        />
```

Comment:
@daniloprates why DialogCloseButton is outside dialog? Can we apply this component to `<Dialog>` component prop like `closeButton={<DialogCloseButton onClose={onClose} />}`